src/prompts/alert/alert.scss

```diff
@@ -0,0 +1,114 @@
+@import '../common.scss';
```

Comment:
Presume we don't end up with duplicated CSS in output declaring dep like this from both modules?

Reply:
Will give a quick check!

Reply:
Was ending up with dups, however have restructured styles and broke up into smaller utility imports which resolves this 👍

Reply:
Oh! Okay, sounds good.